pkg/controller/cincinnati/new.go

```diff
@@ -65,18 +66,22 @@ name = "edge-add-remove"`
 // once up-front makes it MUCH easier to access those resources as-needed
 // throughout the reconciliation code.
 type kubeResources struct {
-	envConfig              *corev1.ConfigMap
-	envConfigHash          string
-	graphBuilderConfig     *corev1.ConfigMap
-	graphBuilderConfigHash string
-	podDisruptionBudget    *policyv1beta1.PodDisruptionBudget
-	deployment             *appsv1.Deployment
-	graphBuilderContainer  *corev1.Container
-	graphDataInitContainer *corev1.Container
-	policyEngineContainer  *corev1.Container
-	graphBuilderService    *corev1.Service
-	policyEngineService    *corev1.Service
-	policyEngineRoute      *routev1.Route
+	envConfig                *corev1.ConfigMap
+	envConfigHash            string
+	graphBuilderConfig       *corev1.ConfigMap
+	graphBuilderConfigHash   string
+	podDisruptionBudget      *policyv1beta1.PodDisruptionBudget
+	deployment               *appsv1.Deployment
+	graphBuilderContainer    *corev1.Container
+	graphDataInitContainer   *corev1.Container
+	policyEngineContainer    *corev1.Container
+	graphBuilderService      *corev1.Service
+	policyEngineService      *corev1.Service
+	policyEngineRoute        *routev1.Route
+	trustedCAConfig          *corev1.ConfigMap
+	pullSecret               *corev1.Secret
+	volumes                  *[]corev1.Volume
```

Comment:
What's your thinking on making these pointers to slices instead of just slices? Usually on a struct it's sufficient to reference a slice directly, since a slice itself is already very similar to a pointer.

Reply:
Just a mistake.  I'm using like it was `[]corev1.Volume`, so it should really just be that.